src/cascadia/TerminalApp/ActionArgs.cpp

```diff
@@ -209,4 +211,29 @@ namespace winrt::TerminalApp::implementation
             return RS_(L"OpenSettingsCommandKey");
         }
     }
+
+    winrt::hstring SetTabColorArgs::GenerateName() const
+    {
+        // "Set tab color to #RRGGBB"
+        // "Reset tab color"
+        if (_TabColor)
+        {
+            til::color c{ _TabColor.Value() };
+            return winrt::hstring{ fmt::format(RS_(L"SetTabColorCommandKey").c_str(), c.ToHexString(true)) };

```

Comment:
does this need the `c_str`? I sorta recall fmt being more resilient/taking anything that is string_viewable

Reply:
okay, no, the thing I did in the Azure connection is to explicitly cast to `std::wstring_view` (`std::wstring_view{ RS_(L"X") }` will suffice)

this is technically cheaper because it doesn't have to round trip through a pointer and re-calculate the entire string length

Reply:
Unfortunately yes, the compiler bites it without it
```

15>C:\Users\migrie\dev\private\OpenConsole\src\cascadia\TerminalApp\ActionArgs.cpp(109,37): error C2672: 'fmt::v6::format': no matching overloaded function found
15>C:\Users\migrie\dev\private\OpenConsole\src\cascadia\TerminalApp\ActionArgs.cpp(109,96): error C2783: 'std::basic_string<Char,std::char_traits<Char>,std::allocator<Char>> fmt::v6::format(const S &,Args &&...)': could not deduce template argument for 'Char'
15>    0 Warning(s)
15>    2 Error(s)
```